extensions/wikia/Chat2/ChatConfig.class.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+class ChatConfig {
+
+	// constants with config file sections
+	const CHAT_DEVBOX_ENV = 'dev';
+	const CHAT_PREVIEW_ENV = 'preview';
+	const CHAT_VERIFY_ENV = 'verify';
+	const CHAT_PRODUCTION_ENV = 'prod';
+
+	const SERVER_TYPE_MAIN = 'Main';
+	const SERVER_TYPE_API = 'Api';
+
+	const VAR_SERVER_BASKET = 'wgChatServersBasket';
+	const VAR_OPERATION_MODE = 'wgChatOperationMode';
+	const CENTRAL_WIKI_ID = 177;
```

Comment:
Let's use `Wikia::COMMUNITY_WIKI_ID` so that we do not copy it all over our code.